utils/imgutils/gif.go

```diff
@@ -214,7 +214,7 @@ func (b *blockReader) close() error {
 }
 
 // decode reads a GIF image from r and stores the result in d.
-func (d *decoder) decode(r io.Reader, configOnly, keepAllFrames bool) error {
+func (d *decoder) decode(r io.Reader, configOnly bool) error {
```

Comment:
@hmhealey - does this change also look good to you?

Reply:
Yeah, that's fine. It might make this a bit more difficult to re-migrade if we wanted to ever update this from upstream, but it's probably no more difficult than it would be with the other changes I already made.